compiler/optimizer/RegisterCandidate.cpp

```diff
@@ -571,89 +571,52 @@ TR_RegisterCandidate::setWeight(TR::Block * * blocks, int32_t *blockStructureWei
    _loadsAndStores = new (comp->trStackMemory()) TR_Array<uint32_t>(comp->trMemory(), numberOfBlocks, true, stackAlloc);
```

Comment:
The commit message needs to be fixed to say "contains old code" instead of "contains old cold".